WorldModel/WorldModel/Core/Languages/EditorEnglish.aslx

```diff
@@ -930,4 +930,6 @@
   <template name="EditorScriptObjectsHide">Hide</template>
   <template name="EditorScriptObjectsPrint">Print</template>
   <template name="EditorScriptObjectsIf">If</template>
+  <template name="EditorGameDisableInGameLog">Disable the in-game log (previously enabled by default)</template>

```

Comment:
This feels confusing in the "Features" tab, a checkbox to disable something... plus a history lesson in brackets. Not saying I know what's better but this feels a bit clunky.

Reply:
Ha, yeah, I agree that it's clunky. I added the history lesson because the control just seems awkward. It's the only box that's ticked by default and disabling rather than enabling. I honestly don't like to add editor controls, but Andy usually liked to add them to make things easier to use. I'll remove the control, and I'll leave the attribute set to true to disable the in-game log (which still seems backwards).

---
Also, I'm trying to avoid saying I'd remove things, but I'd definitely remove the in-game log altogether if it were up to me. We only came up with the idea because they disabled the window, and that's back now.

Reply:
I removed the "Disable the in-game log" control. If someone wanted to use the in-game log, they'd likely be skilled enough to set `game.nohtmllog` to `true`.

I left the "Write log to file" control in there. If you'd rather it not be there, just say the word.

I've left the in-game log in here, too, but I changed it from a JS string variable to an array (as you advised a few months ago).

Reply:
I just uploaded a new commit, and I believe the changes cover all the issues mentioned (and more).

(I'm not sure if GitHub sends a notification when I do that.)

---
> [!NOTE]
> After I closed everything out, I realized this needs to adjust the CSS for the in-game log's dialog window when the player is using a mobile device. I will do this in the morning, and then this should be ready.